Ghost protocol upgrade files/migration-extraction-guide.sh

```
@@ -0,0 +1,368 @@
+# Migration Guide: Extract Memory System to Standalone Service
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)